projects/robots/pal_robotics/tiago++/worlds/tiago++.wbt

```diff
@@ -4,11 +4,10 @@ WorldInfo {
     "This simulation shows a Tiago++ making hello with both arms in an industrial environment. It can also be controlled using the arrows on the keyboard."
   ]
   title "Simulation of PAL Robotics' TIAGo++ robot"
-  coordinateSystem "NUE"
 }
 Viewpoint {
-  orientation 0.28882343295926544 -0.9512999432180493 -0.10774712342777662 5.502273164124365
-  position -0.9802615488886062 1.7304528074815935 2.665124377061189
+  orientation -0.2524577502324584 0.7482692115667424 0.6134804572020622 1.0056342481413238
+  position -14.436996591665887 -9.961169434327116 18.254293081593474
```

Comment:
```suggestion
  orientation -0.04537836798484944 0.015059461718461582 0.9988563542030363 2.5014830680385063
  position 2.956307784608618 -4.303908343005968 1.6938599568355999
```
the viewpoint is not the same than the 2020b version

Reply:
It's not the same, but I prefer the new one :)

Reply:
Actually not :)